internal/domain/usecases/document_usecase.go

```diff
@@ -77,41 +77,29 @@ func (u *documentUsecase) CreateDocument(document *dtos.CreateDocumentDTO) *appe
 }
 
 func (u *documentUsecase) UpdateDocumentByID(ID string, updateMap interface{}) *apperror.AppError {
-	existingDocument, err := u.documentRepository.FindDocumentByID(ID)
-	if err != nil {
+	if err := u.documentRepository.UpdateDocumentByID(ID, updateMap); err != nil {
```

Comment:
if you implemented like this, when user req the doc_id that is not existed in our system. there are no any error in system that means you cannot track any error like this kub, please fix this kub bro @D33102